Assets/BossRoom/Scripts/Server/Game/State/ServerCharSelectState.cs

```diff
@@ -183,6 +189,16 @@ public override void NetworkStart()
 
         private void OnClientConnected(ulong clientId)
         {
+            StartCoroutine(CoroSeatNewPlayer(clientId));
+        }
+
+        private IEnumerator CoroSeatNewPlayer(ulong clientId)
```

Comment:
```suggestion
        private IEnumerator WaitToSeatNewPlayer(ulong clientId)
```

Reply:
done (in my local changes--not taking suggestion here so that I can rename via the IDE).